### PR TITLE
oci: add support for OCI 1.0.0-rc5

### LINKED ISSUE #690: Updates to support OCI spec version v1.0.0-rc5
Review what, if anything, needs to be updated to support spec version `v1.0.0-rc4`.

See:
- https://github.com/opencontainers/runtime-spec/releases/tag/v1.0.0-rc4
- https://github.com/opencontainers/runtime-spec/compare/v1.0.0-rc3...v1.0.0-rc4

### PR DESCRIPTION
**Pull request by devimc**
console size can be specified in config.json this patch
add support to handle this case and change console size
when container is launched

Fixes #690

Signed-off-by: Julio Montes <julio.montes@intel.com>

### CODE REVIEW

**Comment by jodh-intel** [comment]
Fixes #690.

File: src/spec_handlers/process.c [hunk 1]
```diff
@@ -42,6 +42,25 @@ handle_user_section (GNode *root, struct cc_oci_config *config)
...
 }
 
+/*!
+ * function to handle consoleSize section
+ *
+ * \param root contains mount section.
+ * \param config \ref cc_oci_config..
+ */
+static void
+handle_console_size_section (GNode *root, struct cc_oci_config *config) {
```
Comment (jodh-intel) [inline src/spec_handlers/process.c]: This function could do with a check for `config` too.

Comment (devimc) [inline src/spec_handlers/process.c]: oops yes, fixing it and adding tests

**Comment by chavafg** [comment]
qa-passed

**Comment by chavafg** [comment]
qa-passed

**Comment by chavafg** [comment]
qa-failed

**Comment by devimc** [comment]
depends on https://github.com/clearcontainers/hyperstart/pull/15

**Comment by amshinde** [comment]
lgtm

**Comment by chavafg** [comment]
qa-passed

File: tests/data/process-invalid-relative-cwd.json [hunk 2]
```diff
@@ -1,5 +1,9 @@
 {
 	"process": {
+		"consoleSize": {
```
Comment (jodh-intel) [inline tests/data/process-invalid-relative-cwd.json]: Could you add a new `tests/data/process-no-consolesize.json` and add that file to the `spec_handler_test` table in `tests/spec_handlers/process_test.c` to ensure we can tolerate a missing `consoleSize` entry.

Comment (devimc) [inline tests/data/process-invalid-relative-cwd.json]: yes :)

**Comment by jodh-intel** [comment]
lgtm

**Comment by chavafg** [comment]
qa-passed

**Pull request merged**